node/pkg/watchers/near/watcher.go

```diff
@@ -174,7 +174,10 @@ func (e *Watcher) runChunkFetcher(ctx context.Context) error {
 				continue
 			}
 			for _, job := range newJobs {
-				e.schedule(ctx, job, job.delay)
+				err := e.schedule(ctx, job, job.delay)
+				if err != nil {
+					logger.Info("error scheduling transaction processing job", zap.Error(err))
```

Comment:
logger.Error? Same comment at lines 209 and 239.

Reply:
changed it to logger.Error in 211 but Debug in the other two cases and added a reason in the comment: `// Debug-level logging here because it could be very noisy (one log entry for *any* transaction on the NEAR blockchain)`